raddb/mods-available/rest

```diff
@@ -113,6 +113,21 @@ rest {
 	#
 #	multiplex = yes
 
+	#
+	#  fail_body_decode:: Force body decoding to fail for debugging purposes.
+	#
+#	fail_body_decode = no
```

Comment:
These are only available in debug builds and should not be included in the documentation